src/scopes/activityReport/index.test.js

```diff
@@ -625,4 +632,15 @@ describe('filtersToScopes', () => {
         .toEqual(expect.arrayContaining([excludedReport.id, globallyExcluded.id]));
     });
   });
+
+  describe('defaultScope', () => {
+    it('excludes deleted reports', async () => {
```

Comment:
I can't figure out why, but this test is the cause of the timeout failures. Happens reliably on my machine and everything passes perfectly when the test (including the surrounding `describe`) is commented out.